specification/keyvault/resource-manager/readme.md

```diff
@@ -246,3 +246,13 @@ See configuration in [readme.python.md](./readme.python.md)
 ## Java
 
 See configuration in [readme.java.md](./readme.java.md)
+
+## Suppression
+
+``` yaml
+directive:
+  - suppress: IntegerTypeMustHaveFormat
```

Comment:
What is this suppression for?

Reply:
Adding a suppression requires a separate suppression approval through a process, have you already received an approval for this?